doc/source/execute-baselines.rst

```diff
@@ -0,0 +1,48 @@
+Executing Baselines
+===================
+
+Structure
+---------
+
+All baselines are available in the directory `flower/baselines/flwr_baselines <https://github.com/adap/flower/blob/main/baselines>`_. This directory has two different files:
+
+.. code-block:: shell
+
+    - pyproject.toml
+    - requirements.txt
+
+Both files contain all the information about required libraries and their versions. 
+You can install each library separately by using :code: `pip install` or you can use poetry and run code:`poetry install` in the directory where you find the :code:`pyproject.toml` file. After installing all requirements you can start to run your baseline.
+Go to the baseline that you want to execute. The file system is structured that you can first find the paper with their unique identifier such as FedProx represents the paper "Federated Optimization in Heterogeneous Networks". The :code:`fedprox` section contains all available experiments that the paper is covering.   
```

Comment:
```suggestion
Both files contain all the information about required Python packages (libraries, frameworks, ...) and their versions. You can install each library separately by using :code: `pip install` or you can use Poetry and run code:`poetry install` in the directory where you find the :code:`pyproject.toml` file. After installing all requirements, you can start to run your baseline.

Go to the baseline that you want to execute. The directories and files are structured so that you can first find the paper with their unique identifier such that, for example, :code:`FedProx` refers to the paper "Federated Optimization in Heterogeneous Networks". The :code:`fedprox` section contains all available experiments from that paper.
```